src/main/kotlin/com/terraformation/backend/accelerator/api/DeliverablesController.kt

```diff
@@ -362,6 +370,17 @@ data class DeliverablePayload(
   )
 }
 
+data class ImportDeliverableProblemElement(
+    val row: Int,
+    val problem: String,
+) : SuccessResponsePayload
+
+data class ImportDeliverableResponsePayload(
+    val success: Boolean,
+    val problems: List<ImportDeliverableProblemElement> = emptyList(),
+    val message: String? = null,
+) : SuccessResponsePayload
```

Comment:
Could make this extend `ResponsePayload` instead, and then use the `status` field to indicate success or error. As it stands now, this will give you a payload like
```
{
  "status": "ok",
  "success": false,
  ...
}
```
which, while it's not invalid JSON or anything, seems self-contradictory to me. Of course, there's always the argument that this means "the request was successfully processed but the action didn't succeed" but it's no harder to use `status` for this than to add a second field.

That said, if doing it the way I'm suggesting would cause headaches on the FE, ignore this suggestion.